stat/distuv/noncentralt.go

```diff
@@ -0,0 +1,226 @@
+// Copyright ©2025 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package distuv
+
+import (
+	"math"
+	"math/rand/v2"
+
+	"gonum.org/v1/exp/root"
```

Comment:
I'm not comfortable with this.

Reply:
It is indeed weird for production to import experimental packages.
I have moved the helper function into the file itself.